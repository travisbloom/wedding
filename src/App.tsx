import * as React from 'react';
import { ThemeProvider } from 'styled-components';

import { FixedHeader } from './components';
import { Home } from './pages/Home/Home';
import { theme } from './theme';
export class App extends React.Component {
  render() {
    return (
      <ThemeProvider theme={theme}>
        <div>
          <FixedHeader index={0} />
          <Home />
        </div>
      </ThemeProvider>
    );
  }
}
