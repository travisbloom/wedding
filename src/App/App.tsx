import * as React from 'react';
import { HeaderVideo } from 'src/components/HeaderVideo/HeaderVideo';
export class App extends React.Component {
  render() {
    return (
      <div>
        <HeaderVideo />
      </div>
    );
  }
}
