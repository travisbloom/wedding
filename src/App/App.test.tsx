import * as React from 'react';
import { toMatchSnapshot } from '../tests';
import { App } from './App';

it('renders without crashing', () => {
  toMatchSnapshot(<App />);
});
