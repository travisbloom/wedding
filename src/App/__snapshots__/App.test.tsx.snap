// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders without crashing 1`] = `
.c0 {
  position: relative;
  height: 100vh;
  width: 100%;
  overflow: hidden;
}

.c1 {
  height: 100%;
  width: 100%;
  object-fit: cover;
  position: absolute;
  top: 0;
}

.c2 {
  z-index: 10;
  position: relative;
  width: 100%;
  height: 100%;
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
  -webkit-flex-direction: column;
  -ms-flex-direction: column;
  flex-direction: column;
  background-color: rgba(0,0,0,0.5);
  color: white;
}

.c3 {
  font-family: 'Parisienne',cursive;
  margin-top: 80px;
  font-size: 32px;
}

<div
  className="c0"
>
  <video
    autoPlay={true}
    className="c1"
    muted={true}
    playsInline={true}
  >
    <source
      src="/proposal-mobile.mp4"
      type="video/mp4"
    />
    <img
      src="proposal.jpg"
    />
  </video>
  <div
    className="c2"
    onClick={[Function]}
  >
    <h1
      className="c3"
    >
      #BrideAndBloom
    </h1>
  </div>
</div>
`;
